src/app/zap-templates/zcl/data-model/chip/basic-information-cluster.xml

```diff
@@ -14,17 +14,24 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<configurator>
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/service_device_management/BasicInformationCluster.adoc
+Parameters: 
+Git: 0.7-summer-2025-588-g5f6649158
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
   <domain name="CHIP"/>
-
   <struct name="CapabilityMinimaStruct">
     <cluster code="0x0028"/>
-    <item fieldId="0" name="CaseSessionsPerFabric" type="int16u" isNullable="false" writable="false"/>
-    <item fieldId="1" name="SubscriptionsPerFabric" type="int16u" isNullable="false" writable="false"/>
+    <cluster code="0x0039"/>
+    <item fieldId="0" name="CaseSessionsPerFabric" type="int16u" writable="false" default="3" min="3"/>
+    <item fieldId="1" name="SubscriptionsPerFabric" type="int16u" writable="false" default="3" min="3"/>
   </struct>
 
   <enum name="ProductFinishEnum" type="enum8">
     <cluster code="0x0028"/>
+    <cluster code="0x0039"/>
```

Comment:
This comment also seems to still be applicable: https://github.com/project-chip/connectedhomeip/pull/38137#discussion_r2022098070

@hasty, I assume this is intentional by alchemy?

Reply:
I am not sure having the single enum with multiple clusters attached to it is more "right" than having the separate enums.  In particular, the derived cluster _can extend the enum_ in general.  So the enums might happen to match, or they might not, and presupposing they do will break if they start not matching.  @hasty